server/led_sign/led_sign_server.py

```diff
@@ -38,7 +39,9 @@ def WriteCommandToSign(self, request):
             self.proc.kill()
 
         self.proc = subprocess.Popen(command)
-        
+        mock = LedSignMock(self.sign_data)
```

Comment:
by bringing back the argparse logic into this file you can do

`args = parser.parse_args()`

then on this line you can do

```
if args.mock-output:
  # do the mock
```